README.md

```diff
@@ -49,9 +52,44 @@ more flexibility.
 curl -sSL https://raw.githubusercontent.com/soywod/himalaya/master/install.sh | bash
 ```
 
-*See the [wiki section](https://github.com/soywod/himalaya/wiki/Installation)
+Read the Nix section below for a Nix-based installation method. *See the [wiki section](https://github.com/soywod/himalaya/wiki/Installation)
 for other installation methods.*
 
+## Nix
+
+[Nix](https://serokell.io/blog/what-is-nix) users might find this section relevant.
+
+### Installing using Nix
+
+If you already have Nix installed, run the following command to install himalaya:
+
+```bash
+nix-env -if https://github.com/soywod/himalaya/archive/master.tar.gz
+```
+
+If you use Nix with [Flakes](https://nixos.wiki/wiki/Flakes), you can directly run himalaya using:
+
+```bash
+nix run github:soywod/himalaya
```

Comment:
These two commands (`nix-env` and `nix run`) work only after merging this PR. But they can be tested by substituting for the GitHub user and branch name of my fork, viz.:

```bash
nix-env -if https://github.com/srid/himalaya/archive/nixify.tar.gz
```

and,

```bash
nix run github:srid/himalaya/nixify
```

![image](https://user-images.githubusercontent.com/3998/113929144-b539a880-97bd-11eb-8499-cd8544e67b2f.png)